db.js

```diff
@@ -157,27 +258,63 @@ export class ActivityPubDB {
       collectionOrUrl
     )
 
-    const collection = await this.#get(collectionOrUrl)
-
-    for await (const activity of this.iterateCollection(collection)) {
-      await this.ingestActivity(activity, actorId)
+    for await (const activity of this.iterateCollection(collectionOrUrl, { limit: Infinity })) {
+      // Assume newest items will be first
+      const wasNew = await this.ingestActivity(activity, actorId)
+      if (!wasNew) {
+        console.log('Caught up with', actorId || collectionOrUrl)
+        break
+      }
     }
   }
 
-  async * iterateCollection (collection) {
+  async * iterateCollection (collectionOrUrl, { skip = 0, limit = 32 } = {}) {
```

Comment:
We should add a `sort` field which is `1` (default) or -1. When ingesting activities we should do oldest first with `sort: -1`

Reply:
On it.. will push with loadmore